ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    BWoutname = filename[0:-4] + '_BWmask.nrrd'
</pre><p>from pathlib import Path</p><p>Path(filename)</p><p> </p><p>-&gt; remove the extension with this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKIOScanco/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>